style.css

```diff
@@ -0,0 +1,501 @@
+body {
+  font-family: Inter;
+}
+
+.header {
+  background-color: rgb(13, 13, 21, 1);
+  color: rgb(245, 248, 251); 
+}
+
+ul {
+  list-style-type: none;
+}
+.nav__link {
+color: rgb(245, 248, 251);
+font-family: Poppins;
```

Comment:
We also can add font-name to variable. And please  add also alternative font and family